### ISSUE TITLE
Error message: Module version mismatch. Expected 17, got 14.

### ISSUE THREAD

**Issue by ioquatix** [lloeki/svg-preview#7]
Listed below are the incompatible native modules that this package depends on.

pathwatcher@2.0.7
Error message: Module version mismatch. Expected 17, got 14.
runas@1.0.1
Error message: Module version mismatch. Expected 17, got 14.

**Comment by forgot** [comment]
Same here, except seeing:

```console
pathwatcher@2.1.3
Error message: Module version mismatch. Expected 41, got 14.
runas@1.0.1
Error message: Module version mismatch. Expected 41, got 14.
```

**Comment by lloeki** [comment]
I'll fix all of this as soon as I have a spare minute.

**Comment by lloeki** [comment]
Fixed in d44bd833a31b055e6efe5ac5c69898207f649479
